Automattic/wp-calypso
# Delete preemptive domain upsell translation file

#### Proposed Changes

* This PR removes the preemptive domain upsell translation file introduced [here](https://github.com/Automattic/wp-calypso/pull/72296). This file is no longer needed since we now have all the translations.

#### Testing Instructions



* Load this PR
* Verify nothing references this file.
* Ensure the domain upsell still loads correctly.

#### Pre-merge Checklist





Related to #
